assets/css/abstracts/_colors.scss

```diff
@@ -1,18 +1,6 @@
 @import "node_modules/@wordpress/base-styles/colors";
 @import "node_modules/@automattic/color-studio/dist/color-variables";
 
-// @todo remove when `@woocommerce/base-styles` is updated.
-// Greys
-$black: #000;			// Use only when you truly need pure black. For UI, use $gray-900.
-$gray-100: #f0f0f0;		// Used for light gray backgrounds.
-$gray-200: #e0e0e0;		// Used sparingly for light borders.
-$gray-300: #ddd;		// Used for most borders.
-$gray-400: #ccc;
-$gray-600: #949494;		// Meets 3:1 UI or large text contrast against white.
-$gray-700: #757575;		// Meets 4.6:1 text contrast against white.
-$gray-900: #1e1e1e;
-$white: #fff;
```

Comment:
Those variables are defined in recent versions of `@wordpress/base-styles`, so there is no need to define them here too.